テストを実行可能にする

# 目的 : Must
`scripts/execute_unittest.sh`を実行できるようにする

## 背景
`scripts/execute_unittest.sh`を実行すると、以下のエラーが発生している。

```
In file included from ../hrp2/sdk/workspace/hackEV/params/CourseInformationLeft.h:7:0,
                 from ../hrp2/sdk/workspace/hackEV/instances.h:13,
                 from ../hrp2/sdk/workspace/hackEV/EV3Position.cpp:6:
../hrp2/sdk/workspace/hackEV/params/CourseInformation.h:7:20: fatal error: ev3api.h: No such file or directory
 #include "ev3api.h"
                    ^
compilation terminated.
In file included from ../hrp2/sdk/workspace/hackEV/params/CourseInformationLeft.h:7:0,
                 from ../hrp2/sdk/workspace/hackEV/instances.h:13,
                 from ../hrp2/sdk/workspace/hackEV/SpeedCalculator.cpp:6:
../hrp2/sdk/workspace/hackEV/params/CourseInformation.h:7:20: fatal error: ev3api.h: No such file or directory
 #include "ev3api.h"
                    ^
compilation terminated.
```

# やる事 : Must
- [x] `scripts/execute_unittest.sh`の修正

--- a/hrp2/sdk/workspace/hackEV/params/CourseInformation.h
+++ b/hrp2/sdk/workspace/hackEV/params/CourseInformation.h
@@ -4,7 +4,7 @@
  */
 #pragma once
 
-#include "ev3api.h"
+#include "product.h"
 #include "EV3Position.h"
 #include <vector>
 

